nucleus/common/internal-api/src/main/java/fish/payara/internal/notification/admin/BaseSetNotifierConfigurationCommand.java

```diff
@@ -95,6 +96,9 @@ public abstract class BaseSetNotifierConfigurationCommand<C extends PayaraNotifi
     @Param(name = "noisy", optional = true, defaultValue = "true")
```

Comment:
I didn't notice it on the other PR, but similar to my other commend we should probably remove the default value here too so that we don't constantly get warned about the deprecated parameter even when we don't specify it

```
CLI031: Warning: Option "noisy" is obsolete and will be ignored.
```